Doctrine/EventSubscriber.php

```diff
@@ -64,7 +66,12 @@ public function persistObject(DataEvent $event)
         }
 
         $objectManager->persist($event->getData());
-        $objectManager->flush();
+        try {
+            $objectManager->flush();
+        } catch (\Exception $e) {
```

Comment:
ahh nevermind it will require to rename `$event`:

```php
public function persistObject(DataEvent $dataEvent)
{
    // ...
    $event = Events::FLUSH_ERROR;
    $objectManager->persist($event->getData());

    try {
        $objectManager->flush();
        $event = Events::POST_CREATE;
    } finally {
        $this->eventDispatcher->dispatch($event, $dataEvent);
    }

    // this point won't be reached if an exception has been thrown
    $event->stopPropagation();
```